lib/genesis.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Configurable genesis.
+ *
+ * @module
+ */
+
+'use strict';
+
+const rb = require('crypto').randomBytes;
```

Comment:
I think we should consider writing genesis JSON into file. There may be a method to write it somewhere to the 'data' folder for later use.

As option, it could be something like this:

```JavaScript
const fs = require('fs');

Genesis.prototype.writeToFile = function writeToFile(path) {
    fs.writeFileSync(path, JSON.stringify(this, null, 4));
}
```

Suggested is just an example. Exact implementation may differ.